doc/whatsnew/fragments/6543.feature

```diff
@@ -0,0 +1,3 @@
+Update ``pyreverse`` to differentiate between aggregations and compositions.
```

Comment:
I think it would be nice to add a short summary of how we distinguish between the two here.

Reply:
Updated fragment :+1: